Tasks/HelmInstallerV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -0,0 +1,14 @@
+{
+  "loc.friendlyName": "Helm tool installer",
+  "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=851275)",
+  "loc.description": "Install Helm on agent machine.",
+  "loc.instanceNameFormat": "Install Helm $(helmVersion)",
+  "loc.input.label.helmVersion": "Helm Version Spec",
+  "loc.input.help.helmVersion": "Specify the version of Helm to install",
+  "loc.messages.DownloadStableVersionFailed": "Failed to read latest kubectl version from stable.txt. From URL %s. Using default stable version %s.",
```

Comment:
Should we remove kubectl related messages?

Reply:
removed messages